features/support/apiREST.js

```diff
@@ -14,13 +14,11 @@ ApiREST.prototype.init = function (world) {
 ApiREST.prototype.disconnect = function () {};
 
 ApiREST.prototype.pathApi = function (path) {
-  var basePath = '/api';
-
-  if (apiVersion) {
-    basePath += '/' + apiVersion;
-  }
+  return config.url + '/api/1.0/' + path;
```

Comment:
Forgot it, i read it too fast and didn't notice that we were within feautres/ ;)